src/components/voucher/VoucherCollectionTabs.tsx

```diff
@@ -16,13 +16,24 @@ const VoucherCollectionTabs: React.VFC<{ vouchers: VoucherProps[] }> = ({ vouche
   }[] = [
     {
       key: 'available',
-      name: formatMessage(commonMessages.status.available),
-      isDisplay: voucher => !!voucher.available,
+      name: formatMessage(usersMessages.tab.available),
+      isDisplay: voucher => voucher.available,
+    },
+    {
+      key: 'notYet',
+      name: formatMessage(usersMessages.tab.notYet),
+      isDisplay: voucher =>
+        (voucher.startedAt && voucher.startedAt.getTime() > Date.now() && !voucher.status.used) || false,
     },
     {
       key: 'unavailable',
```

Comment:
NITS: could be expired, in conformity wit coupon

Reply:
Actually this code copy from coupon XD